parm/config/config.efsoi

```diff
@@ -0,0 +1,37 @@
+#!/bin/ksh -x
+
+########## config.efsoi ##########
+# Ensemble update specific, dependency config.anal
+
+echo "BEGIN: config.efsoi"
+
+# Get task specific resources
+. $EXPDIR/config.resources efsoi
+
+export npe_enkf=$npe_eupd
+
+
+# Use NAM_ENKF below for serial EnKF
+##export NAM_ENKF="analpertwtnh=0.9,analpertwtsh=0.9,analpertwttr=0.9"
```

Comment:
This should be in a branching structure that turns on when serial, rather than relying on users to uncomment and comment blocks of code.

Reply:
Lines removed